website/intro/index.html.markdown

```diff
@@ -3,79 +3,42 @@ layout: "intro"
-guide for all available features as well as internals.
-
-> **Hands-on:** Try the [Get Started](https://learn.hashicorp.com/tutorials/terraform/infrastructure-as-code) tutorials on HashiCorp Learn.
-
-## What is Terraform?
-
-Terraform is a tool for building, changing, and versioning infrastructure
-safely and efficiently. Terraform can manage existing and popular service
-providers as well as custom in-house solutions.
-
-Configuration files describe to Terraform the components needed to
-run a single application or your entire datacenter.
-Terraform generates an execution plan describing
-what it will do to reach the desired state, and then executes it to build the
-described infrastructure. As the configuration changes, Terraform is able
-to determine what changed and create incremental execution plans which
-can be applied.
-
-The infrastructure Terraform can manage includes
-low-level components such as
-compute instances, storage, and networking, as well as high-level
-components such as DNS entries, SaaS features, etc.
+Below, HashiCorp co-founder and CTO Armon Dadgar describes how Terraform can help solve common infrastructure challenges.
 
 <iframe src="https://www.youtube.com/embed/h970ZBgKINg" frameborder="0" allowfullscreen="true"  width="560" height="315" ></iframe>
 
-Examples work best to showcase Terraform. Please see the
-[use cases](/intro/use-cases.html).
 
-The key features of Terraform are:
+
+## Key Features
 
 ### Infrastructure as Code
 
-Infrastructure is described using a high-level configuration syntax. This allows
-a blueprint of your datacenter to be versioned and treated as you would any
-other code. Additionally, infrastructure can be shared and re-used.
+Infrastructure is described using a high-level [configuration language](/docs/language/index.html) in human-readable, declarative configuration files. This allows you to create a blueprint that can be versioned, shared, and reused.
 
 ### Execution Plans
 
-Terraform has a "planning" step where it generates an _execution plan_. The
-execution plan shows what Terraform will do when you call apply. This lets you
-avoid any surprises when Terraform manipulates infrastructure.
+Terraform generates an _execution plan_ describing what it will do and asks for your approval before creating, updating, or destroying infrastructure. This allows you to review changes before they are applied.
 
 ### Resource Graph
 
-Terraform builds a graph of all your resources, and parallelizes the creation
-and modification of any non-dependent resources. Because of this, Terraform
-builds infrastructure as efficiently as possible, and operators get insight into
-dependencies in their infrastructure.
+Terraform builds a resource graph and parallelizes the creation and modification of any non-dependent resources. This allows Terraform to
+build resources as efficiently as possible and gives operators greater insight into their infrastructure.
```

Comment:
```suggestion
build resources as efficiently as possible and gives you greater insight into your infrastructure.
```

Reply:
Done! I wasn't sure about this one because I went back and forth about whether the audience might be broader somehow because this is an overview page...but then I realized that it's likely operators who are this deep in reference docs. Thank you!